src/components/Navigation.js

```diff
@@ -18,111 +18,103 @@ import {
   UNLEASH_TOKEN_BALANCES_FEATURE_FLAG,
 } from '../constants';
 
-class Navigation extends React.Component {
-  constructor(props) {
-    super(props);
+function Navigation(props) {
```

Comment:
suggestion(non-blocking): I'd suggest making each of the props explicit, to make it clear what must be passed to this component.
```suggestion
function Navigation({ history }) {
```

In this specific case, making `history` explicit will allow us easily modify it to use the following syntax as soon as we upgrade `react-router-dom` to `v5.x`:
```js
import { useHistory } from "react-router-dom";

function Navigation() {
  const history = useHistory();
  ...
}
```

Reply:
Done https://github.com/HathorNetwork/hathor-explorer/pull/275/commits/05763f6ce5c4f4705d25ddb6b4a3cbb92419adc5